web/components/modals/ChatModal/ChatModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { Modal } from 'antd';
+import { FC } from 'react';
+import { ErrorBoundary } from 'react-error-boundary';
+
+import styles from './ChatModal.module.scss';
+
+import { ComponentError } from '../../ui/ComponentError/ComponentError';
+import { ChatContainer } from '../../chat/ChatContainer/ChatContainer';
+import { ChatMessage } from '../../../interfaces/chat-message.model';
+import { CurrentUser } from '../../../interfaces/current-user';
+import { UserDropdown } from '../../common/UserDropdown/UserDropdown';
+
+export type ChatModalProps = {
+  messages: ChatMessage[];
+  currentUser: CurrentUser;
+  handleClose: () => void;
+};
+
+export const ChatModal: FC<ChatModalProps> = ({ messages, currentUser, handleClose }) => {
+  if (!currentUser) {
+    return null;
+  }
+  const { id, displayName, isModerator } = currentUser;
+
+  const modalBodyStyle = {
+    padding: '0px',
+    height: '55vh',
+  };
+
+  return (
+    <ErrorBoundary
+      // eslint-disable-next-line react/no-unstable-nested-components
+      fallbackRender={({ error, resetErrorBoundary }) => (
+        <ComponentError
+          componentName="ChatModal"
+          message={error.message}
+          retryFunction={resetErrorBoundary}
+        />
+      )}
+    >
+      <Modal
+        open
+        centered
+        maskClosable={false}
+        footer={null}
+        title={<UserDropdown />}
+        maskStyle={{
+          zIndex: 700,
+          background:
+            'linear-gradient(180deg, rgba(0,0,0,0) 40%, rgba(0,0,0,0.45) 50%, rgba(0,0,0,0.5) 100%)',
```

Comment:
If we don't want to be able to click outside the modal to close it, then I think we should still have the backdrop cover the entire background. Otherwise buttons on the top half of the page look like they're clickable. 

I know we do want to prioritize seeing the video while chatting, but at least you can still see it! ..and know that the other buttons around it might not be clickable.

So if we remove that line it might feel better?